scripts/mobr_sensitivity.R

```diff
@@ -1,6 +1,7 @@
 # Sensitivity test for Mobr functions
 # using Felix May's MoBspatial package
-library(MoBspatial)
+.libPaths("C://Users//Xiao//Documents//R//win-library//3.2")
```

Comment:
Please remove any absolute paths here at when writing files. The reason is so that users can reproduce the results without having to change the code. Rstudio makes managing paths (including the R package library) and git a lot easier.